lib/models/user.js

```diff
@@ -89,14 +186,29 @@ User.set('toObject', {
     delete ret.deactivator;
     delete ret.resetter;
     delete ret.pendingHashPass;
-    delete ret.bytesUploaded;
+    // delete ret.paymentProcessors;
```

Comment:
I think these may have been already uncommented before, these should probably not be on the toObject for user?